packages/ripple-nuxt-tide/lib/module.js

```diff
@@ -161,6 +161,7 @@ const nuxtTide = function (moduleOptions) {
 
   // transpile auth modules
   this.options.build.transpile.push(/@nuxtjs\/auth-next/)
+  this.options.build.transpile.push(/requrl/)
```

Comment:
This is sub dependency of auth-next. I debugged by disabling preview module and started to transpile its dependencies to verify the offending one.